packages/phone-number-privacy/src/index.ts

```diff
@@ -1,6 +1,9 @@
+import { BLINDBLS } from 'bls12377js-blind'
 import * as functions from 'firebase-functions'
+import { PHONE_NUMBER_PRIVACY_SECRET_KEY } from './config'
 
 export const getSalt = functions.https.onRequest((request, response) => {
-  console.log(request.body)
-  response.send(request.body)
+  const privateKey = new Buffer(PHONE_NUMBER_PRIVACY_SECRET_KEY)
```

Comment:
Nit, have any needing processing of the key happen in config.ts so other files can just consume the key

Reply:
i'll leave as is for now. another file could also want the raw key rather than the buffer, so we can always extract this logic to the config based on how it ends up being used